services/jenkins/jenkins-coverage.tester.js

```diff
@@ -1,176 +1,27 @@
 'use strict'
 
-const { ServiceTester } = require('../tester')
 const { isIntegerPercentage } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
 
-const t = new ServiceTester({
-  id: 'jenkins-coverage',
-  title: 'JenkinsCoverage',
-  pathPrefix: '/jenkins',
-})
-module.exports = t
+// The below page includes links to various publicly accessible Jenkins instances
+// although many of the links are dead, it is is still a helpful resource for finding
+// target Jenkins instances/jobs to use for testing.
+// https://wiki.jenkins.io/pages/viewpage.action?pageId=58001258
 
```

Comment:
I've updated the tests to use different Jenkins instances that seem to be much more performant and stable than the Apache instance. That gives us good test targets to validate both jacoco and cobertura coverage formats, so I've removed several of the mock tests accordingly (I believe we were mocking due to the lack of consistently available live jobs)